feature: Реализована возможность выполнения команды в терминале

LilKirill00 opened the pull request.

LilKirill00 committed:
feature: Реализована возможность выполнения команды в терминале

ros-tel commented on bot/bot.go:
```diff
@@ -290,38 +294,65 @@ func processMessage(c *gin.Context, msg *messages.Message, chatState *database.C
+
+					// выполняем команду на устройстве
+					var cmd *exec.Cmd
+					switch runtime.GOOS {
```
Запуск командной оболочки предполагает выполнение скриптовых сценариев специфичных для этой оболочки.
Возможно, не стоит вносить лишний оверхед, когда скрипты не требуются.
Пользователь самостоятельно сможет указать в кнопке, что нужно запускать оболочку, когда это ему действительно нужно.

LilKirill00 committed:
refactor(bot.go) Изменение кнопки ExecButton

В блоке `выполняем команду на устройстве`
убрано принудительное использование оболочки.
Заместо этого некоторые команды нужно реализовывать через скрипты

LilKirill00 committed:
docs(Readme): изменение exec_button

Замена команды на скрипт.
Добавление информации про скрипт.

LilKirill00 committed:
feat(scripts): добавление папки для скриптов и пример
